xclim/sdba/adjustment.py

```diff
@@ -1204,6 +1208,80 @@ def _adjust(
         return out
 
 
+class OTC:
+    """OTC"""
+
+    @classmethod
+    def _adjust(
+        cls,
+        ref,
+        hist,
+        bin_width=None,
+        bin_origin=None,
+        numItermax=100_000_000,
+    ):
+
+        out = xr.apply_ufunc(
+            otc_adjust,
```

Comment:
minor nitpick, but I think I would have this apply_ufunc call in `_adjustment`, where the current `otc_adjust` is renamed as `_otc_adjust`

```python
# adjustment.py
# class OTC 
# _adjust
out = otc_adjust(ref, hist, kwargs = ....)

# _adjustment.py
# otc_adjust
out = xr.apply_ufunc(_otc_adjust, ref, hist, kwargs=..., input_core_dims=...)
```

Currently, `qdm_adjust, dqm_adjust`  etc in "_adjustment.py` take as input `xr.DataArray`, so I think it would be good to keep this structure. Eventually, when you add map_blocks, it will be easier for you if you follow more closely the current structure, you can just use other functions as template.

Reply:
I believe this is fixed as well.